### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by Hatins** [uzh-rpg/ssms_event_cameras#5]
Hi @NikolaZubic 

Hello, recently I carefully studied the paper and source code. There are a few aspects I don't quite understand. 

Firstly, in the process of Output masking, why is only 'C' masked and not 'A' and 'B'? 

Secondly, in your code, if only adjusting the 'step_scale', the mask of 'C' remains unaffected.
```python
            step = step_scale * torch.exp(self.log_step)

            freqs = step / step_scale * self.Lambda[:, 1].abs() / (2 * math.pi)
            mask = torch.where(freqs < bandlimit * 0.5, 1, 0)  # (64, )
```
Since the freqs are not linked with the value of step_scale. (freqs = step / step_scale * self.Lambda[:, 1].abs() / (2 * math.pi) -> freqs =  torch.exp(self.log_step) * self.Lambda[:, 1].abs() / (2 * math.pi)). Is the code wrong?

Question 2 is accompanied by another question, Question 3. In the paper, it's only mentioned that generalization from low frequency to high frequency is achieved by masking 'C', but in the code, the discretization of 'A' and 'B' is also related to 'step_scale'. So, I wonder, for generalization from low frequency to high frequency, is it necessary to adjust all three values 'A', 'B', and 'C'?

```python
       if not torch.is_tensor(step_scale) or step_scale.ndim == 0:
            # step_scale = torch.ones(signal.shape[-2], device=signal.device) * step_scale
            step = step_scale * torch.exp(self.log_step)
```
